exporters/memory/test/in_memory_span_data_test.cc

```diff
@@ -0,0 +1,33 @@
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <gtest/gtest.h>
+
+using opentelemetry::exporter::memory::InMemorySpanData;
+using opentelemetry::sdk::trace::Recordable;
+using opentelemetry::sdk::trace::SpanData;
+
+TEST(InMemorySpanData, AddRecordable)
+{
+  InMemorySpanData data(100);
+
+  ASSERT_EQ(0, data.GetSpans().size());
+
+  opentelemetry::nostd::span<std::unique_ptr<SpanData>> recordables(
+      new std::unique_ptr<SpanData>(new SpanData()), 1);
+
+  for (auto &recordable : recordables)
```

Comment:
Do we need this loop, when the above span only contains one recordable?

Reply:
Nope, I've removed that